pkg/prom/diagnostics.go

```diff
@@ -78,13 +78,14 @@ var diagnosticDefinitions map[string]*diagnosticDefinition = map[string]*diagnos
 		Label:       "Kubecost metrics available",
 		Description: "Determine if metrics from Kubecost are available during last 5 minutes.",
 	},
-	NodeExporterDiagnosticMetricID: {
-		ID:          NodeExporterDiagnosticMetricID,
-		QueryFmt:    `absent_over_time(node_cpu_seconds_total{%s}[5m] %s)`,
-		Label:       "Node-exporter metrics available",
-		Description: "Determine if metrics from node-exporter are available during last 5 minutes.",
-		DocLink:     fmt.Sprintf("%s#node-exporter-metrics-available", DocumentationBaseURL),
-	},
+	// Commented out because it is not critical to Opencost metrics and can cause confusion with false alarms about node-exporter
```

Comment:
I'm curious why we comment out instead of removing? My opinion has always been remove as we have git commit history if we need to revert or look at something.
CC: @mbolt35 thoughts?

Reply:
sure removing it :+1: